core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,29 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    abort-all-workflows-activity {
+      depends-on = [service-unbind]
+      timeout = 1 hour
+    }
+    
+    # This phase is the one responsible for flushing all outstanding write requests to the Database
+    stop-io-activity {
+      depends-on = [service-stop]
+      timeout = 30 seconds
+    }
+  }
 }
 
 system {
   # If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
-  #abort-jobs-on-terminate = false
+  abort-jobs-on-terminate = false
+  
+  # If 'true', a SIGINT will trigger Cromwell to attempt to gracefully shutdown in server mode, in particular clearing up all 
```

Comment:
This is actually a mistake it should say `SIGTERM`. I saw this in the coordinated shutdown docs that made me wary `By default, the CoordinatedShutdown will be run when the JVM process exits, e.g. via kill SIGTERM signal (SIGINT ctrl-c doesn’t work)`.
However I'm not sure why they say that because looking at the code they just add a normal JVM shutdown hook.